rustls/src/lib.rs

```diff
@@ -10,20 +10,19 @@
 //! * IMPORT AS USUAL FROM `rustls`: `use rustls;` OR `use rustls::...`
 //!
 //! <!-- TODO CLEAN UP THIS DOCUMENTATION -->
+//! <!-- TODO: ADDRESS HOW TO BUILD WITH A CRYPTO PROVIDER ON A TARGET WITH NO ATOMIC PTR -->
 //! THIS FORK SUPPORTS using `Arc` from `portable-atomic-util` to support targets with no atomic ptr, with the following requirements:
 //! * USE Rust nightly toolchain
 //! * USE `--cfg portable_atomic_unstable_coerce_unsized` in RUSTFLAGS FOR `cargo build` (etc.)
 //! * USE `--cfg unstable_portable_atomic_arc` in RUSTFLAGS FOR `cargo build` (etc.)
-//! * NEED TO ADD MULTIPLE DEPENDENCIES WITH SPECIFIC FEATURES ENABLED WHEN BUILDING FOR A TARGET WITH NO ATOMIC PTR, AS DESCRIBED DIRECTLY BELOW
+//! * NEED TO ADD MULTIPLE DEPENDENCIES WITH SPECIFIC FEATURES ENABLED WHEN BUILDING FOR A TARGET WITH NO ATOMIC PTR, AS DESCRIBED ALMOST DIRECTLY BELOW
+//! * ALSO NEED TO BUILD WITH A CRYPTO PROVIDER FOR THIS CRATE TO BE USEFUL IN GENERAL (may try building with a built-in crypto provider enabled, if possible for the desired target)
```

Comment:
I may move this statement back to where it came from, would like to revisit & cleanup another time